grapher/axis/Axis.ts

```diff
@@ -306,40 +372,10 @@ abstract class AbstractAxis {
         )
     }
 
-    // calculates coordinates for ticks, sorted by priority
-    @computed private get tickPlacements(): TickPlacement[] {
-        return sortBy(this.baseTicks, (tick) => tick.priority).map((tick) => {
-            const bounds = Bounds.forText(
-                this.formatTick(tick.value, {
-                    isFirstOrLastTick: tick.isFirstOrLastTick,
```

Comment:
Can get rid of `isFirstOrLastTick` in `formatValue`, too. It's no longer being used.

Reply:
Awesome, thanks!